tracks/rust/exercises/clock/mentoring.md

```diff
@@ -7,7 +7,14 @@
 ### Reasonable solutions
 
 A reasonable solution should do the following:
-`items needed here`
+
+* Use `#[derive]`
+* Implement `fmt::Display`
+* Reuse logic by implementing `Clock::add_minutes` in terms of `Clock::new`, or vice versa.
+* Prefer `24 * 60` over the rather magical `1440`.
+* Use `%` instead of a loop.
+* Have clean logic for performing the modulus.
+* (more items needed here)
 
 ### Examples
```

Comment:
We need to change ### Examples to #### Examples since it's a subsection of "Reasonable solutions".

With that, I would prefer the constant be `MINUTES_PER_DAY` in the example, just because it reads clearer.